main.nf

```diff
@@ -18,10 +18,26 @@ nextflow.enable.dsl = 2
 
 WorkflowMain.initialise(workflow, params, log)
 
-include {METAGENOMICS_SR} from './workflows/metagenomics-sr'
+if (params.platform == 'illumina') {
+    include { ILLUMINA } from './workflows/illumina'
+} else if (params.platform == 'nanopore') {
+    include { NANOPORE } from './workflows/nanopore'
+}
+
+
+workflow ARCADIASCIENCE_METAGENOMICS {
```

Comment:
is it necessary to have this be `ARCADIASCIENCE_METAGENOMICS` instead of `METAGENOMICS`? it seems a bit redundant, but I can't tell if this was a design decision

Reply:
not necessary, I tried to follow the convention that Mert/Austin have in phylorthology because the nf-core templates now append NFCORE_PIPELINENAME so it was easier to sub in ARCADIASCIENCE here.